modules/ai-agents/pages/mcp-server/quickstart.adoc

```diff
@@ -18,9 +18,7 @@ This enables natural-language automation for internal workflows, using your own
 You'll need the following to complete this quickstart:
 
 - xref:ROOT:get-started:rpk-install.adoc[Redpanda CLI (`rpk`)]
-- At least version 4.56.0 of Redpanda Connect
-+
-If you need to upgrade, see xref:get-started:upgrade/rpk-upgrade.adoc[].
+- Redpanda Connect 4.56.0 or later. If you need to upgrade, see xref:get-started:upgrade/rpk-upgrade.adoc[].
```

Comment:
but this could just be a link, similar to the one above, like: 
- xref:get-started:upgrade/rpk-upgrade.adoc[Redpanda Connect 4.56.0 or later]